source/Command.cpp

```diff
@@ -221,6 +221,70 @@ bool Command::HasConflict() const
 
 
 
+// Load this command from an input file (for testing or scripted missions).
+void Command::Load(const DataNode &node)
+{
+	for(int i = 1; i < node.Size(); ++i)
+	{
+		const string &input = node.Token(i);
+		
+		if(input == "menu")
```

Comment:
Will there ever be an input that requires setting multiple commands, or requires disambiguation? If not, (and perhaps even if "maybe"), we should use a map instead of a large `if-else if` chain.

Reply:
Replaced here and in the split-off PR #4997 that just handles Command-loading.